### DISCUSSION TITLE
LinkToAnotherRecord `Display Field` [Aug 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3189] [Discord Threads]
> 🙋‍♂️ **CaptainKranch**  
## Hello, I have a relational db that is not been displayed properly in nocodb. is this normal?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1005970007110459454/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/661905455894888493/1005970007529893959/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@177618286756757507>    
#### If you meant display value - consider changing `primary value` as per your need. You can find details here   
####    
#### https://docs.nocodb.com/setup-and-usages/primary-value  
#  
> 🙋‍♂️ **CaptainKranch**  
#### Great!! thanks a lot  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1005970008192598070)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
